lib/src/jiffy.dart

```diff
@@ -597,8 +617,4 @@ class Jiffy {
       (year % 4 == 0) && ((year % 100 != 0) || (year % 400 == 0));
 
   bool get isLeapYear => _isLeapYear(_dateTime.year);
-
-  static bool isJiffy(var input) => input is Jiffy;
-
-  static bool isDateTime(var input) => input is DateTime;
```

Comment:
@AadumKhor Instead of deleting these methods right away, add `@Deprecated` tag for people to migrate since they are public methods.